plugins/module_utils/foreman_helper.py

```diff
@@ -1017,6 +1019,39 @@ def __init__(self, **kwargs):
         super(ForemanTaxonomicEntityAnsibleModule, self).__init__(foreman_spec=foreman_spec, **kwargs)
 
 
+class ForemanDataStreamModule(ForemanTaxonomicEntityAnsibleModule):
+    def __init__(self, **kwargs):
+        foreman_spec = dict(
+            original_filename=dict(type='str'),
+            scap_file=dict(type='path'),
+        )
+        foreman_spec.update(kwargs.pop('foreman_spec', {}))
+        super(ForemanDataStreamModule, self).__init__(foreman_spec=foreman_spec, **kwargs)
+
+    def datastream_run(self):
```

Comment:
Why don't you override the actual 'run' method?

Reply:
because I'm a lazy bum and hate Python's `super`.